code/include/env/mcause.h

```diff
@@ -0,0 +1,10 @@
+#ifndef RISCV_MCAUSE_H
+#define RISCV_MCAUSE_H
+
+#define MCAUSE_EXCEPTION_SHIFT 0
+#define MCAUSE_INTERRUPT_SHIFT 31
+
+#define MCAUSE_EXCEPTION_MASK 0xFFFFFFF1ull
+#define MCAUSE_INTERRUPT_MASK 0x80000000ull
+
```

Comment:
also add the values for the exception and interrupt causes, given in table 8.6.1 in task #27